### DISCUSSION TITLE
POST ​/v3​/bets​/place Place bet request Place Bet ERROR

### DISCUSSION THREAD

**Discussion by jshelton31** [cloudbet/docs#66] [General]
Hello, 

I have been trying to run the API support execute test for placing a bet, but getting an error every time. 
https://www.cloudbet.com/api/?urls.primaryName=Trading#/Trading/PlaceBet

Just running the standard test info they have

{
  "acceptPriceChange": "BETTER",
  "currency": "PLAY_EUR",
  "eventId": "18148915",
  "marketUrl": "soccer.total_goals/over?total=3",
  "price": "1.86",
  "referenceId": "0e81877b-4b10-4587-a4cf-89f4336ac843",
  "side": "BACK",
  "stake": "1.1"
}

and no response even with a valid key. 

Can someone provide a sample CURL req that works so I can proceed. Thank you.

**Comment by kgravenreuth** [comment]
Hi @jshelton31 

Thanks for raising this issue here. 

Upon checking your payload, it seems you are using the `"referenceId": "0e81877b-4b10-4587-a4cf-89f4336ac843"` which is a non-unique reference ID as it was already used in our [examples protobuf](https://github.com/Cloudbet/docs/blob/c3024e079db1d64684f742fbf7e26663aa3d8d36/cloudbet/trading.proto#L14) in this repo.

Could you please retry with a unique UUID which you can generate programatically or [online](https://www.uuidgenerator.net/version4)? Note that every new bet placement request must have a unique UUID in the `referenceId` field.

Please let us know if this resolves your issue.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
